### ISSUE TITLE
Crash

### ISSUE THREAD

**Issue by llamafilm** [bitfocus/companion-module-qsys-remote-control#9]
It's my first time trying to use this plugin and it crashes immediately before I have a chance to enter any config.'
I'm running 3.0.0 on macOS

![Screenshot 2023-03-30 at 10 20 36 AM](https://user-images.githubusercontent.com/3991046/228915679-4e888fe3-cc0d-4f39-b9cf-8726a0181e26.png)

```
system: ** Connection started **
Starting up module class: i
Sentry disabled
Module-host accepted registration
error: Trace: Init failed: TypeError: Cannot read properties of undefined (reading 'has')
    at /Applications/Companion.app/Contents/Resources/bundled-modules/qsys-remote-control/main.js:2:192440
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async /Applications/Companion.app/Contents/Resources/bundled-modules/qsys-remote-control/main.js:2:172337
error: Failed to init: Error: Cannot read properties of undefined (reading 'has') TypeError: Cannot read properties of undefined (reading 'has')
    at i.addControl (/Applications/Companion.app/Contents/Resources/bundled-modules/qsys-remote-control/main.js:2:233222)
    at /Applications/Companion.app/Contents/Resources/bundled-modules/qsys-remote-control/main.js:2:217163
    at Array.forEach (<anonymous>)
    at i.initVariables (/Applications/Companion.app/Contents/Resources/bundled-modules/qsys-remote-control/main.js:2:217145)
    at i.configUpdated (/Applications/Companion.app/Contents/Resources/bundled-modules/qsys-remote-control/main.js:2:216870)
    at i.init (/Applications/Companion.app/Contents/Resources/bundled-modules/qsys-remote-control/main.js:2:216602)
    at /Applications/Companion.app/Contents/Resources/bundled-modules/qsys-remote-control/main.js:2:192382
    at /Applications/Companion.app/Contents/Resources/bundled-modules/qsys-remote-control/main.js:2:172176
    at t.default._tryToStartAnother (/Applications/Companion.app/Contents/Resources/bundled-modules/qsys-remote-control/main.js:2:171223)
    at /Applications/Companion.app/Contents/Resources/bundled-modules/qsys-remote-control/main.js:2:172383
system: ** Connection stopped **
```

**Comment by brionco** [comment]
I am having the same problem.  Mac and Windows, 3.0 beta and 3.1 experimental.

Still works fine in 2.4.2

**Comment by satyrux** [comment]
Hello I confirm that the problem is still existing I had to use version 2.4.2
The version for the raspberry pi has the same problem I had to use version 2.2.3

**Comment by onfire4g05** [comment]
I think this may have been fixed, but just hadn't been merged. Can anyone that was having issues please test it again?

**Comment by llamafilm** [comment]
I just tried the latest companion develop branch (Build 3.0.0-rc1+5909-master-725c3583) which uses Qsys module 2.0.1 and it has the same problem.
I see there's a newer 2.0.2 release from 2 days ago so I'm not sure why that's not included in develop?

**Comment by onfire4g05** [comment]
You'll need to get the latest beta.

**Comment by llamafilm** [comment]
Can you clarify what you mean by "latest beta"?  As I said I'm already on the develop branch of companion, which I thought was newer than beta branch.

**Comment by onfire4g05** [comment]
If you're seeing 2.0.1, it isn't the latest version. I just downloaded the [latest beta from the website](https://user.bitfocus.io/builds/companion, 3.0.0+6009, and it has v2.0.2 of this module in it.

**Comment by satyrux** [comment]
I just tested with the Beta and the problem seems to be fixed in any case on the windows version.
Many thanks for the support and the correction of the problem :-)

**Comment by onfire4g05** [comment]
Awesome! Sorry about the issue in RC1. I'll go ahead and close this issue.

**Comment by llamafilm** [comment]
My bad, I forgot to run `yarn update` after pulling from git.  Confirmed, this bug is fixed now in 2.0.2.
